mordhau.kvp

```diff
@@ -20,6 +20,7 @@ App.RootDir=./mordhau/
 App.BaseDirectory=./mordhau/629800/
 App.ExecutableWin=629800\Mordhau\Binaries\Win64\MordhauServer-Win64-Shipping.exe
 App.ExecutableLinux=629800/Mordhau/Binaries/Linux/MordhauServer-Linux-Shipping
+App.SteamWorkshopDownloadLocation={{$FullBaseDir}}Mordhau/Plugins
```

Comment:
Since this doesn't use the Steam Workshop section, this should be blanked out or removed entirely. If this is kept blanked out, the version needs bumped on `Meta.MinAMPVersion`.

Reply:
@IceOfWraith is there a way to hide the Steam Workshop section completely considering it does not use them?

and when you say blanked out, you mean
`App.SteamWorkshopDownloadLocation=` without anything after it?

Whichever way this works, I will make the amendment and bump the version

Reply:
There's no way to hide it. I've asked Mike for that multiple times with no memorable response.. It's very misleading and clutter. Just another UX thing that should be improved and more thought out.

The way you changed it there would be fine. Then bump the version to 2.4.3.0 and it should be good to go.

Reply:
Done and done